storage/replica_command.go

```diff
@@ -1595,7 +1597,14 @@ func (r *Replica) VerifyChecksum(
 			p()
 		} else {
 			// Replication consistency problem!
-			log.Fatalf("replica: %s, checksum mismatch: e = %x, v = %x", r, args.Checksum, c.checksum)
+
+			// shouldPanic is set to false when the env var is unset, or fails parsing.
+			shouldPanic, _ := strconv.ParseBool(os.Getenv("COCKROACH_PANIC_ON_CONSISTENCY_FAILURE"))
```

Comment:
Shouldn't this be load in the server context?
https://github.com/cockroachdb/cockroach/blob/master/server/context.go
See readEnvironmentVariables()